subgraphs/abracadabra/src/mapping.ts

```diff
@@ -36,7 +36,7 @@ export function handleLogAddCollateral(event: LogAddCollateral): void {
   let depositEvent = new Deposit(event.transaction.hash.toHexString() + "-" + event.transactionLogIndex.toString());
   let market = getMarket(event.address.toHexString());
   let cauldronContract = cauldron.bind(event.address);
-  let collateralToken = getOrCreateToken(Address.fromString(market.inputTokens[0]));
+  let collateralToken = getOrCreateToken(Address.fromString(market.inputTokens[INT_ZERO]));
```

Comment:
umm, you don't need to use constant for this...

Reply:
so even if inputTokens is a list value? That means I could just put 'market.inputTokens' ?